Bottom: a51751a95b4577784235feed0abd949c4180cc86
Top:    c70c04e8a04e111637bb7668b5b4b1983c1664b4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 20:52:51 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 7980090..97f92a2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
